test/surface_test.py

```diff
@@ -1198,6 +1198,21 @@ def test_convert_alpha_SRCALPHA(self):
         finally:
             pygame.display.quit()
 
```

Comment:
- [ ] line 16 can be removed completely
- [ ] lines 771: 
```
        s.get_view(as_unicode("2"))
        s.get_view(as_bytes("2"))
```
change to: 
`        s.get_view("2")`